datadog/dogstatsd/container.py

```diff
@@ -13,31 +14,51 @@ class UnresolvableContainerID(Exception):
     """
 
 
-class ContainerID(object):
+class Cgroup(object):
     """
-    A reader class that retrieves the current container ID parsed from a the cgroup file.
+    A reader class that retrieves either:
+    - The current container ID parsed from the cgroup file
+    - The cgroup controller inode.
 
     Returns:
-    object: ContainerID
+    object: Cgroup
 
     Raises:
         `NotImplementedError`: No proc filesystem is found (non-Linux systems)
         `UnresolvableContainerID`: Unable to read the container ID
     """
 
     CGROUP_PATH = "/proc/self/cgroup"
+    DEFAULT_CGROUP_MOUNT_PATH = "/sys/fs/cgroup"  # default cgroup mount path.
```

Comment:
Since it isn't customizable, there is no point calling it the "default".

```suggestion
    CGROUP_MOUNT_PATH = "/sys/fs/cgroup"  # cgroup mount path.
```